spec/lib/heroku_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Heroku do
+  subject { described_class }
+
+  before do
+    allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new(environment))
+  end
+
+  describe '.enabled?' do
+    let(:environment) { 'development' }
+
+    context 'in development' do
+
+      it 'returns true if `RUNNING_IN_HEROKU` is present' do
+        expect(ENV).to receive(:[]).with("RUNNING_IN_HEROKU").and_return("TRUE")
+
+        expect(described_class.enabled?).to be true
+      end
+      it 'returns false otherwise' do
+        expect(ENV).to receive(:[]).with("RUNNING_IN_HEROKU").and_return(nil)
```

Comment:
Is this supposed to have `.with("RUNNING_IN_HEROKU")`?

Reply:
I would say so. If the environment variable is not set then it always should return false?

Reply:
I was confused as I didn't know that `expect` can also implicitly behave like an `allow` as well as assert. Thanks for clarifying!